classroom.rb

```diff
@@ -0,0 +1,12 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+  end
+end
```

Comment:
- [ ] Kindly add `attr_reader: students` to your classroom class to create a getter method for the instance variable @students. This means that after adding the attribute reader, you can access the student's array from outside the class and retrieve its value
- [ ]  The same case also happens to your student.rb file. Kindly add `attr_reader: classroom`. I would also suggest that you add a method that serves as `the 'belongs-to' side (a student belongs to a classroom)` in your student class. I am highlighting this here because you did not make any changes in your student class. Here is how it should look like

```ruby
class Student < Person
  attr_reader :classroom

  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
end
```